rocks.kfs.CampusEngagementEmailSend/Jobs/CampusEngagementEmailSend.cs

```diff
@@ -0,0 +1,152 @@
+            if ( errorMessages.Any() )
+            {
+                StringBuilder sb = new StringBuilder();
+                sb.AppendLine();
+                sb.Append( string.Format( "{0} Errors: ", errorCount ) );
+                errorMessages.ForEach( e => { sb.AppendLine(); sb.Append( e ); } );
+                string errors = sb.ToString();
+                context.Result += errors;
+                var exception = new Exception( errors );
+                HttpContext context2 = HttpContext.Current;
+                ExceptionLogService.LogException( exception, context2 );
+                throw exception;
+            }
+        }
+
+        private void SendEmailToCampusPastors()
+        {
+            var rockContext = new RockContext();      
+            var campuses = rockContext.Campuses.ToList();
+
+            foreach( Campus campus in campuses )
+            {
+                var pastor = campus.LeaderPersonAlias.Person;
+                var email = pastor.Email;   
+
+                var groupService = new GroupService( rockContext );
+                List<Group> allCampusGroups = new List<Group>();
+
+                foreach( GroupType groupType in groupTypes )
+                {
+                    List<Group> groups = new List<Group>( groupService.GetByGroupTypeId( groupType.Id ) ).Where( g => g.CampusId == campus.Id ).ToList();
+                    allCampusGroups.AddRange( groups );
+                }
+                
+                   
+                if ( email.IsNotNullOrWhiteSpace() )
+                {
+                    //groupsNotified.Add( group.Id );
+
+                    var mergeObjects = Rock.Lava.LavaHelper.GetCommonMergeFields( null, pastor.IsNotNull() ? pastor : null );
+                    mergeObjects.Add( "Person", pastor.IsNotNull() ? pastor : null );
+                    mergeObjects.Add( "Groups", allCampusGroups );
+                    mergeObjects.Add( "Campus", campus );
+
+                    var recipients = new List<RecipientData>();
+                    recipients.Add( new RecipientData( email, mergeObjects ) );
+
+                    var emailMessage = new RockEmailMessage( systemEmailGuid );
+                    emailMessage.SetRecipients( recipients );
+                    var errors = new List<string>();
+                    emailMessage.Send( out errors );
+
+                    if ( errors.Any() )
+                    {
+                        errorCount += errors.Count;
+                        errorMessages.AddRange( errors );
+                    }
+                    else
+                    {
+                        engagementEmailsSent++;
+                    }
+                }
+                else
+                {
+                    errorCount += 1;
+                    errorMessages.Add( string.Format( "No email specified for {0} and no fallback email provided.", pastor.FullName ) );
```

Comment:
Since there is no fallback email in this job let's remove that statement from the error.
```suggestion
                    errorMessages.Add( string.Format( "No email specified for {0}.", pastor.FullName ) );
```